docs/PayPalNativeCheckout/README.md

```diff
@@ -11,10 +11,10 @@ Follow these steps to add PayPal Native Checkout payments:
 ## Setup a PayPal Developer Account
 
 You will need to set up authorization to use the PayPal Payments SDK.
-Follow the steps in [Get Started](https://developer.paypal.com/api/rest/#link-getstarted) to create a client ID and secret, to generate access tokens.
+Follow the steps in [Get Started](https://developer.paypal.com/api/rest/#link-getstarted) to create a client ID and secret, needed to generate an access token.
 
-You will need a server integration to create an order to capture funds using the [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2).
-For initial setup, the `curl` commands below can be used as a reference for making server-side RESTful API calls.
+You will need a server-side integration to create an order to capture funds using the [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2).
+This order will be used to authorize or capture funds. For initial setup, the `curl` commands below can be used as a reference for making server-side RESTful API calls.
```

Comment:
```suggestion
The order created on your server will be used to authorize or capture funds. For initial setup, the `curl` commands below can be used as a reference for making server-side RESTful API calls.
```
We should specify where the funds are authorized or captured after `funds` above. In the SDK? Also on their server?